gensim/summarization/bm25.py

```diff
@@ -4,7 +4,7 @@
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
```

Comment:
Please remove `Keras-2.1.2-py2.7.egg from PR (comment here, because I can't add comment to binary file)